### ISSUE TITLE
Invoke-UDEndpoint in Textbox OnEnter doesn't work

### ISSUE THREAD

**Issue by TurnItOff-OnAgain** [ironmansoftware/powershell-universal#2139]
### Steps to Reproduce

Invoke-UDEndpoint does not "click" a button

Link for forum post

https://forums.ironmansoftware.com/t/cant-enable-a-radio-button-other-bugs-found/8740/3?u=turnitoff_onagain

` New-UDTextbox -id "tb_Hostname" -Label "Hostname" -OnEnter {
    Invoke-UDEndpoint -id "btn_Get"
}`

`new-udhtml -Markup "<br>"`

`New-UDButton -id "btn_Get" -Text "Get computers" -OnClick {
    Show-UDToast -Message "clicked"
}`

### Expected behavior

Should "click" the button and show the UDToast

### Actual behavior

![image](https://user-images.githubusercontent.com/26926284/221954929-cfa0de98-538d-419f-956c-ddaf6b9da559.png)

The button is not clicked. I tried tying the button to another button to see if it was a textbox issue and it has the same results

### Environment data

3.7.13

Windows Server 2019

**Comment by donlent** [comment]
Dear all
Is it possible to take care of this in one of the next releases?
All over sudden it stopped working and actually it is quite annoing not being able to work with the Enter button anymore... :-(

Thanks a lot!

**Comment by donlent** [comment]
Hey @adamdriscoll 
Sorry to bother but would it be possible to fix this in the next release? Maybe it's just a typo thing in your code or something like that because the Enter Key gets triggered but it somehow cannot resolve the Element IDs properly...

Would really appreciate since we're using this functionality quite often in our projects.

Thanks in advance!

**Comment by TurnItOff-OnAgain** [comment]
Hey @adamdriscoll is this still on the roadmap to get fixed? The typo in the error was fixed, but the code itself doesn't work. I copied this right from the docs

https://docs.powershelluniversal.com/apps/components/inputs/textbox#onenter

```
New-UDTextbox -OnEnter {
    Invoke-UDEndpoint -Id 'submit'
}

New-UDButton -Id 'submit' -OnClick {
    Show-UDToast -Message 'From Textbox'
}
```
and get this error

![image](https://github.com/user-attachments/assets/6654b285-bee6-49c2-937b-eef357a7f7c2)

**Comment by adamdriscoll** [comment]
I'll move it into the next release. I haven't looked into it.

**Comment by adamdriscoll** [comment]
I was able to get this to work using the -Session parameter on Invoke-UDEndpoint. 

```
New-UDTextbox -OnEnter {
    Invoke-UDEndpoint -Id 'submit' -Session
}

New-UDButton -Id 'submit' -OnClick {
    Show-UDToast -Message 'From Textbox'
}
```

It's not clear to me why I added this switch. If you don't specify it, it doesn't include the necessary IDs in the lookup for the endpoint to invoke. Unfortunately, that was done in 2020 so I'll need to go back and figure that out. I just don't really know how this would ever work without that switch being used.
